### PR TITLE
Add app stores CGUs and developer policies

### PR DESCRIPTION
**Pull request by LucasVerneyDGE**
Hi,

Here is a work in progress to add CGUs from app stores. FIY, here is a quick list of the relevant policies I found so far (not sure all of them are fully relevant, I should double check).

Google Play Store :
- [x] Conditions d’utilisation : https://play.google.com/intl/fr_fr/about/play-terms/index.html
- [x] Contrat de distribution : https://play.google.com/intl/ALL_fr/about/developer-distribution-agreement.html
- [x] Privacy policy : https://policies.google.com/privacy?hl=fr-FR 
- [x] Brand guidelines : https://developer.android.com/distribute/marketing-tools/brand-guidelines
- [x] Review guidelines : https://support.google.com/googleplay/android-developer/answer/9904549
- [x] Google terms : https://policies.google.com/terms?hl=fr-FR
 
Apple App Store :
- [ ] Contrat de dévelopeur : https://developer.apple.com/terms/apple-developer-agreement/Apple-Developer-Agreement-French.pdf 
- [x] Privacy policy : https://www.apple.com/legal/privacy/fr-ww/ 
- [x] Review guidelines : https://developer.apple.com/app-store/review/guidelines/ 

Best,

### CODE REVIEW

**Comment by LucasVerneyDGE** [comment]
Related question before writing these providers, how should history be handled?

For instance, Play Store terms have an archived version available online at https://play.google.com/intl/fr_fr/about/play-terms/eea/archive/. What is the best workflow for you to (manually?) include this version in CGUs-data prior to including the provider for continuous update?

@MattiSG also told me about a previous tool which had some archived content. Do you have more infos about this tool, to eventually merge history for these providers?

Thanks!

**Comment by Ndpnt** [comment]
Hi @LucasVerneyDGE, thanks for your interest on this project !
Regarding your question, we do not have yet decided how we will handle this, so I do not have an answer for now.

File: providers/Apple.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "serviceProviderName": "Apple",
+    "documents": {
+        "privacy": {
+            "url": "https://www.apple.com/legal/privacy/fr-ww/",
```
Comment (LucasVerneyDGE) [inline providers/Apple.json:5]: Here (and below): Use US / English versions.

**Comment by LucasVerneyDGE** [comment]
This one should be rebased and ready for review. I'm letting the remaining one on the side for now (and opening a dedicated issue to discuss PDF).

Fwiw, the Google Play Developer Policy is having an update effective on August 12: https://support.google.com/googleplay/android-developer/answer/9904549?hl=en vs https://support.google.com/googleplay/android-developer/answer/9914283. Might be a good test case :)

**Comment by LucasVerneyDGE** [comment]
P.S.: For the archived versions (https://play.google.com/intl/fr_fr/about/play-terms/eea/archive/ for instance), I can do a manual import / PR on the data repository, if you want.

File: services/Apple.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Apple",
+    "documents": {
+        "privacy": {
+            "location": "https://www.apple.com/legal/privacy/en-ww/",
+            "contentSelector": "main[role=main]"
```
Comment (MattiSG) [inline services/Apple.json:6]: > I'm not sure we need the attribute selector 🙂

File: services/AppleAppStore.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Apple App Store",
+    "documents": {
+        "reviewGuidelines": {
+            "location": "https://developer.apple.com/app-store/review/guidelines/",
+            "contentSelector": "main[role=main]"
```
Comment (MattiSG) [inline services/AppleAppStore.json:6]: > I'm not sure we need the attribute selector 🙂

File: services/Apple.json (hunk 2 above)
Comment (LucasVerneyDGE) [reply services/Apple.json]: Fixed

File: services/AppleAppStore.json (hunk 3 above)
Comment (LucasVerneyDGE) [reply services/AppleAppStore.json]: Fixed

**Comment by LucasVerneyDGE** [comment]
See https://github.com/ambanum/CGUs-data/pull/3 for the import of the archived version.

**Pull request merged**